content/en/dora_metrics/setup/incidents.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Sending Incident Events for DORA Metrics
+kind: documentation
+description: Learn how to send incident events for DORA Metrics.
+aliases:
+- /continuous_integration/dora_metrics/setup/incidents
+is_beta: true
+further_reading:
+- link: "https://app.datadoghq.com/release-notes?category=Software%20Delivery"
+  tag: "Release Notes"
+  text: "Check out the latest Software Delivery releases! (App login required)"
+- link: "/tracing/service_catalog"
+  tag: "Documentation"
+  text: "Learn about the Service Catalog"
+- link: "/code_analysis"
+  tag: "Documentation"
+  text: "Learn about Code Analysis"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">DORA Metrics is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="https://forms.gle/Eqq6uXfGjYxmqpjDA" header="false" >}}
+DORA Metrics private beta is closed. Fill out the form below to be added to the waitlist.
+{{< /callout >}}
+
+## Overview
+
+Incident events are used in order to compute Change Failure Rate and Mean Time to Restore.
+To send incident events, use the [DORA Metrics API][1]. The following attributes are required:
+
+- At least one between `services` and `team`
+- `started_at`
+
+You can optionally add the following attributes to the incident events:
+- `finished_at` for *resolved incidents*. This attribute is required for [time to restore service](#time-to-restore-service).
+- `id` for identifying incidents when they are created and resolved. This attribute is user-generated; when not provided, the endpoint returns a Datadog-generated UUID.
+- `name` to describe the incident.
+- `severity`
```

Comment:
The severity can be any value? Is there a list of supported severities?

Reply:
It can be any value right now